tox.ini

```diff
@@ -63,25 +62,15 @@ deps = readme_renderer
 commands = python setup.py check --restructuredtext --strict
 
 [testenv:ci]
-basepython = python3.8
-passenv = LANG TERM LANGUAGE LC_ALL LD_LIBRARY_PATH BRANCH INTEGRATION_TESTS_REPO INTEGRATION_TESTS_BRANCH
 setenv =
     API_CLIENT_RUN_ON_REDIS = true
-deps =
-    pip-tools
-    coverage
 allowlist_externals =
-    git
-    /bin/ln
-    /bin/rm
+    {[testenv:integrationtests]allowlist_externals}
 commands_pre =
     python -c "import fcntl; fcntl.fcntl(1, fcntl.F_SETFL, 0)"
-    pip install -e .
-    pip install -r requirements/dev.txt
-    git clone -b {env:INTEGRATION_TESTS_BRANCH:master} {env:INTEGRATION_TESTS_REPO:git@github.com:gridsingularity/gsy-backend-integration-tests.git} {envtmpdir}/gsy-backend-integration-tests
-    ln -sf {envtmpdir}/gsy-backend-integration-tests/gsy_e_sdk_tests/integration_tests/ ./integration_tests
+    {[testenv:integrationtests]commands_pre}
 commands =
-    pytest unit_tests
-    behave ./integration_tests/ --stop
+    {[testenv:unittests]commands}
+    {[testenv:integrationtests]commands} --stop
```

Comment:
Are you sure this works? What happens if we add a new line to the `testenv:integrationtests` commands?

Reply:
Yes! I tested it locally before pushing and also the ci checks passed. 
In [this line](https://github.com/gridsingularity/gsy-e-sdk/runs/5810656597?check_suite_focus=true#step:8:399) you can see how it looks like. :)

Reply:
That's nice :) But I'm a bit afraid that this will break if we add a new line in `testenv:integrationtests`, because at that point the `--stop` argument will be appended to the wrong command.

Reply:
If the theoretical new command is added before the actual execution of `behave`, this should not break. But I do get your point. Adapted on last commit.